components/Body.js

```diff
@@ -15,7 +15,7 @@ const StyledBody = styled.div`
   border-radius: 10px;
   box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
   width: 90vw;
-  height: 75vh;
+  min-height: 70vh;
```

Comment:
It's not necessary to wrap your `StyledBody` into a separate `Body` component which does nothing. You could export your styled component directly.
Regarding naming: `StyledBody` actually renders a `<div>` and not a `<body>` tag. I think it would be clearer to call it `PageContainer` or something like that...